crates/libs/sys/Cargo.toml

```diff
@@ -21,6 +21,7 @@ path = "../targets"
 
 [features]
 default = []
+doc = []
```

Comment:
How about docs, plural? I recognize this is temporary so not a blocker. But I may grumble something in the hallway when we walk past each other.

Reply:
Wouldn't want that! It could go either way. I based it on the `doc` attribute. 🤷‍♂️

Reply:
Switched to `docs`.